test/parallel_api/algorithm/alg.nonmodifying/transform_if.pass.cpp

```diff
@@ -211,11 +213,15 @@ test_inplace()
 int
 main()
 {
-    test<::std::int32_t>();
-    test<::std::int64_t>();
-
-    test_inplace<::std::int32_t>();
-    test_inplace<::std::int64_t>();
+    test<std::int8_t>();
```

Comment:
1) We are increasing n up to 100 times..

Reply:
Yes, there will be some increase in test time with this patch which I have been estimating through CI. It is necessary to increase `n` to test the large submitter path.